src/octoprint/plugins/action_command_notification/__init__.py

```diff
@@ -120,6 +124,11 @@ def action_command_handler(self, comm, line, action, *args, **kwargs):
             return
 
         message = parameter.strip()
+
+        for filter in self._filters:
+            if filter.fullmatch(message):
```

Comment:
Hm... I'm pretty sure that in other places we use `match` behaviour (so, partial match). I think we should do that here as well for reasons of consistency with other filter settings in OctoPrint. The user can require a full match by using `^` and `$` markers in their patterns.